### ISSUE TITLE
some question for loss

### ISSUE THREAD

**Issue by lvwuyunlifan** [jeong-tae/RACNN-pytorch#14]
hi, @jeong-tae :
I follow your model to train. The number of pre-apn trainings is 2000, and it seems strange to see the loss of the cls and the loss of the rank, as shown. 

![4f](https://user-images.githubusercontent.com/27217235/49203182-488a9400-f3e2-11e8-8963-facdce3c8f75.png)
![168](https://user-images.githubusercontent.com/27217235/49203183-49232a80-f3e2-11e8-8e46-ba98c00bec94.png)

Also accuracy has been very low, no more than 1. What is the reason for this?

**Comment by jeong-tae** [comment]
this is because APN predict wrong location. I didn’t find how to train APN correctly yet.
There are some missing details...and I think nobody knows about that. If you can find pretrained APN or code/detail param, let me know.

**Comment by lvwuyunlifan** [comment]
Oh, thank you. If I know the code/details about APN I will tell you.
